src/remote-docker/remote-docker.yml

```diff
@@ -51,7 +51,7 @@ commands:
                 exit 0
               fi
             else
-              printf "Required environment variables `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` unavailable. Reverting back to Circle CI docker.\n"
+              printf "Required environment variables \`AWS_ACCESS_KEY_ID\` and \`AWS_SECRET_ACCESS_KEY\` unavailable. Reverting back to Circle CI docker.\n"
```

Comment:
wonder if the backticks were meant as delimiters in which case i feel it's better to just use single-quotes which doesn't require escape.

Reply:
yeah I wasn't sure about the newline character at the end. It looks like it's ok to be in single quotes, so I updated it here.